### ISSUE TITLE
Add gender as metadata to MP speeches

### ISSUE THREAD

**Issue by fredrik1984** [welfare-state-analytics/riksdagen-corpus#27]
It should be possible to sort MP speeches by gender (as well as year, party belonging and chamber). I guess this will also require that we have the MP person names to derive gender from some kind of name index?

**Comment by ninpnin** [comment]
Yes, this can be done by simply adding gender to the MP database. You would indeed need a list of Swedish names by gender, this is what I found https://barnnamn.net/flicknamn-a-o/ https://barnnamn.net/pojknamn-a-o/ ... Better sources are appreciated!

**Comment by fredrik1984** [comment]
Excellent source haha! But maybe something to begin with

**Comment by MansMeg** [comment]
I agree. I think we can use those list:
1. Remove names that occur in both list (ambigious)
2. Map with the MEPs first names.
3. Those that are not mapped, just send us a list of the MEPs and we will fix it manually

**Comment by fredrik1984** [comment]
Sounds like a good plan to me

**Comment by ninpnin** [comment]
https://github.com/welfare-state-analytics/riksdagen-corpus/blob/dev/db/mp/metadata/names.csv

**Comment by MansMeg** [comment]
Cool! BTW we should rename the MP csv to members_of_parliament.csv instead since in the long run, we want to expand this corpus to other time frames.

**Comment by ninpnin** [comment]
I was able to infer the gender of 8612/10556 MPs with this method.

**Comment by fredrik1984** [comment]
Good job! I also guess that the method missed "double names", like Sven-Bertil Anna-Maria etc? And surely MPs with names not don't have a typical Swedish origin

**Comment by ninpnin** [comment]
Here's the MP database with currently available gender information: https://github.com/welfare-state-analytics/riksdagen-corpus/blob/dev/db/mp/members_of_parliament.csv

**Comment by MansMeg** [comment]
Great! Could you send me an Fredrik a list with the names without somewhere we can fix the last ones?

**Comment by MansMeg** [comment]
Has this been done by Felix now?

**Comment by ninpnin** [comment]
Yes, but not merged yet. Status in #66

**Comment by MansMeg** [comment]
Ah! Great!
